examples/00-mapdl-examples/BodyFlux-Averaging.py

```diff
@@ -0,0 +1,324 @@
+"""
+.. _ref_solenoid_magnetostatic_2d:
+
+=======================================
+Analysis of a 2D Magnetostatic Solenoid
+=======================================
+
+This example shows how to gather and plot results with material discontinuities
+across elements (Power graphics style) versus the default full average results
+(Full graphics style).
+
+
+Description
+===========
+
+Mechanical APDL has two averaging methods depending the method used for
+presenting results:
+
+* **Full graphics** presents the entire selected model with node averaged results.
+  In the case of an node shared by two or more elements that have differing
+  materials, the stress field is continuous across the element material boundary
+  (the shared nodes).
+
+* **Power graphics** presents the entire selected model with averaged results
+  within elements of the same material, and discontinuous across material
+  boundaries.
+
+There are other differences between the two methods; this example concentrates
+on material boundaries.
```

Comment:
```suggestion
This example focuses on material boundaries.
```
Shouldn't this be after the example title below? It and the next sentence can be part of the same paragraph.

Reply:
You are right!

Reply:
I think I'm going to just remove the below title.